third_party/java_deps/BUILD.gn

```diff
@@ -24,3 +24,7 @@ java_prebuilt("annotation") {
 java_prebuilt("json") {
   jar_path = "artifacts/json-20220924.jar"
 }
+
+java_prebuilt("kotlin-stdlib") {
+  jar_path = "artifacts/kotlin-stdlib-1.3.31.jar"
```

Comment:
Kotlin is up to 1.8.x at this time: https://mvnrepository.com/artifact/org.jetbrains.kotlin/kotlin-stdlib 

Can we use the newest version? There have been several major changes since 1.3. The latest stable version is `1.8.10`.

Reply:
Yes, we can, I chose version 1.3.31 because the default kotlin version in Ubuntu 22.04 is 1.3.31.

I will post another PR to update the kotlinc version in cloud-build-java to 1.8.10, and update the build instructions to prevent user from building kotlin app with the default kotlinc come with the Ubuntu 22.04.